pandas/core/dtypes/dtypes.py

```diff
@@ -848,7 +848,7 @@ def __setstate__(self, state) -> None:
 
 
 @register_extension_dtype
-class PeriodDtype(PandasExtensionDtype):
+class PeriodDtype(dtypes.PeriodPseudoDtype, PandasExtensionDtype):
```

Comment:
not a huge fan of PeriodPseuodDtype name

Reply:
im open to suggestions.  joris objected to calling it PeriodDtype

Reply:
this is basically a cython PeriodBaseDtype

Reply:
Updated with new name + green